journal/original/01-journal.adoc

```diff
@@ -388,8 +388,9 @@ Then again, when Adam transgressed the law and commandment of God,
 "`the Lord God called unto Adam, and said unto him.
 Where art thou?
 And he said, I heard thy voice in the garden, and I was afraid.
+because I was naked,`" Gen. 3:10-9. 
```

Comment:
Probably here let's fix the period on line 390 as well. I'm OK leaving the quotations "as is" because this is the original version.

Reply:
gotcha! thanks!